code/modules/networks/computer3/mainframe2/misc_terms.dm

```diff
@@ -1037,7 +1037,7 @@
 
 #define DISARM_CUTOFF 10 //Can't disarm past this point! OH NO!
 
-	mats = 80 //haha this is a bad idea
+	mats = list("POW-3"=113, "MET-3" = 134, "CON-2" = 117, "DEN-3" = 142) //haha this is a bad idea
```

Comment:
POW-3 and MET-3, that's basically just all cerenkite in this recipe

Reply:
Cerenkite does not meet the requirements of tier 3 metal. Though it meeting tier 3 power is something I didn't know.